cli-messaging/src/test/java/com/salesforce/messaging/EventKeyTest.java

```diff
@@ -0,0 +1,126 @@
+package com.salesforce.messaging;
```

Comment:
Basically a copy of `pmd-cataloger/src/test/java/sfdc/sfdx/messaging/EventKeyTest.java`.